testapi.pm

```diff
@@ -835,21 +835,28 @@ Setting C<$no_regex> will cause it to do a plain string search.
 =cut
 
 sub wait_serial {
+    my $regexp = shift;
+    my %args   = OpenQA::Isotovideo::Utils::compat_args(
+        {
+            regexp           => $regexp,
+            timeout          => 90,
```

Comment:
~~Why not `$bmwqemu::default_timeout` again?~~ I wonder if $bmwqemu::default_timeout should be increased and used here instead (But that's ouside the scope of this pr)

Reply:
The important thing with this line is that `undef` is mapped to default value. But the old implementation also mapped `0` to default value.